lib/microkube/renderer.rb

```diff
@@ -0,0 +1,49 @@
+require 'openssl'
+require 'pathname'
+require 'yaml'
+require 'base64'
+
+module MicroKube
+  # Renderer is class for rendering MicroKube templates.
+  class Renderer
+    TEMPLATE_PATH  = Pathname.new('./templates')
+
+    JWT_KEY = 'config/secrets/barong.key'.freeze
```

Comment:
can we avoid this, it's application specific, and should only be a variable somehow.